WordPress/Classes/ViewRelated/Reader/ReaderCardDiscoverAttributionView.swift

```diff
@@ -43,7 +43,7 @@ import WordPressShared
         if contentProvider?.sourceAttributionStyle() == SourceAttributionStyle.Post {
             configurePostAttribution(contentProvider!)
         } else if contentProvider?.sourceAttributionStyle() == SourceAttributionStyle.Site {
-            configureSiteAttribution(contentProvider!)
+            configureSiteAttribution(contentProvider!, verboseAttribution: false)
```

Comment:
Nitpicky is good!  :)

This can be tested via the Discover feed.  Find a post that has a site attribution (says "Visit ___) and tap to view the post (don't click on the attribution).  The detail should show a snippet from the site, and show the attribution again, but this time saying "Visit ___ for more".